src/frontend/apps/impress/src/hook/useSupport.tsx

```diff
@@ -0,0 +1,43 @@
+import { Crisp } from 'crisp-sdk-web';
+import { useEffect } from 'react';
+
+import { User } from '@/core';
+
+const isCrispConfigured = (): boolean => {
+  return typeof window !== 'undefined' && !!window.$crisp;
+};
+
+export const initializeSupportSession = (user: User) => {
+  if (!isCrispConfigured()) {
+    return;
+  }
+  Crisp.setTokenId(user.id);
```

Comment:
I am not sure anymore of this part … let's discuss it before merging, I am reviewing their code